<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewOrderWindow</class>
 <widget class="QDialog" name="NewOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#NewOrderWindow

{
background-image: url(:/Images/Stillder UI/BuilderInterface.png);
}</string>
  </property>
  <widget class="QListWidget" name="OrderList">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>201</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/Images/Stillder UI/LoginMenu/blank.png);
background-image: url(:/Images/Stillder UI/LoginMenu/blank.png);

</string>
   </property>
  </widget>
  <widget class="QComboBox" name="MenuSelect">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>161</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/Images/Stillder UI/LoginMenu/blank.png);
background-image: url(:/Images/Stillder UI/LoginMenu/blank.png);

</string>
   </property>
  </widget>
  <widget class="QListWidget" name="MenuList">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>491</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/Stillder UI/LoginMenu/blank.png);
background-image: url(:/Images/Stillder UI/LoginMenu/blank.png);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="MenuName">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>9</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 117, 255);
font: 100 25pt &quot;Bahnschrift&quot;;
background: transparent;</string>
   </property>
   <property name="text">
    <string>MenuName</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SelectedMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(0,0,0);
font: 63 10pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>selected menu : Mclays Foods</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TotalOrderPrice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(0,0,0);
font: 63 10pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>Total order price : </string>
   </property>
  </widget>
  <widget class="QLabel" name="NoOfItems">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(0,0,0);
font: 63 10pt &quot;Bahnschrift&quot;;</string>
   </property>
   <property name="text">
    <string>Number of items : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>120</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#Save{
background-image: url(:/Images/Stillder UI/SaveButton/normal.png);
border: none;
}
#Save:hover
{
background-image: url(:/Images/Stillder UI/SaveButton/hover.png);
}

#Save:pressed
{
background-image: url(:/Images/Stillder UI/SaveButton/clicked.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>1133</x>
     <y>3</y>
     <width>61</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Close
{
background-image: url(:/Images/Stillder UI/CrossButton/normal.png);
border: none;

}
#Close:hover
{
background-image: url(:/Images/Stillder UI/CrossButton/hover.png);

}
#Close:pressed
{
background-image: url(:/Images/Stillder UI/CrossButton/clicked.png);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="Sort">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/Images/Stillder UI/LoginMenu/blank.png);
background-image: url(:/Images/Stillder UI/LoginMenu/blank.png);

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Remove">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>120</y>
     <width>45</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#Remove{
color: rgb(161, 117, 255);
font: 100 25pt &quot;Bahnschrift&quot;;
background-image: url(:/Images/Stillder UI/AddAndRemove/Remove/normal.png);
border: none;
background-repeat: none;
}
#Remove:hover
{
background-image:  url(:/Images/Stillder UI/AddAndRemove/Remove/hover.png);
}

#Remove:pressed
{
background-image:  url(:/Images/Stillder UI/AddAndRemove/Remove/clicked.png);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>45</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#Add{
color: rgb(161, 117, 255);
font: 100 25pt &quot;Bahnschrift&quot;;
background-image: url(:/Images/Stillder UI/AddAndRemove/Add/normal.png);
border: none;
background-repeat: none;
}
#Add:hover
{
background-image:  url(:/Images/Stillder UI/AddAndRemove/Add/hover.png);
}

#Add:pressed
{
background-image:  url(:/Images/Stillder UI/AddAndRemove/Add/clicked.png);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectSort">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#SelectSort{
background-image: url(:/Images/Stillder UI/SortButton/normal.png);
border: none;
}
#SelectSort:hover
{
background-image: url(:/Images/Stillder UI/SortButton/hover.png);
}

#SelectSort:pressed
{
background-image: url(:/Images/Stillder UI/SortButton/clicked.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
